pkg/kubelet/network/dns/dns.go

```diff
@@ -99,6 +100,14 @@ func omitDuplicates(strs []string) []string {
 func (c *Configurer) formDNSSearchFitsLimits(composedSearch []string, pod *v1.Pod) []string {
 	limitsExceeded := false
 
+	for i, search := range composedSearch {
+		if len(search) > utilvalidation.DNS1123SubdomainMaxLength {
+			composedSearch = composedSearch[:i]
+			limitsExceeded = true
+			break
```

Comment:
this should be a continue if we want to have all valid "searches" https://github.com/kubernetes/kubernetes/pull/100651/commits/a974fb5f8883506d1456907d23d26ca0195120ce#r635548677

Reply:
no need to continue since there is no more code after, just removing it

Reply:
I think you are commenting on outdated code.

Reply:
yeah, I just realised, sorry